azure-devops/golang/stages/40-delivery/arm.yaml

```diff
@@ -46,11 +46,25 @@ stages:
               overrideParameters: '-location $(location) -environment $(environment) -functionAppName $(functionAppName) -inboundAllowedIps $(inboundAllowedIps)'
               templateLocation: 'Linked artifact'
               deploymentMode: 'Incremental'
+              deploymentOutputs: 'OUTPUTS'
           - task: 'AzureFunctionApp@1'
             inputs:
               appName: '$(environment)-fn-$(functionAppName)-$(location)'
               appType: 'functionAppLinux'
               azureSubscription: '$(serviceConnection)'
               package: '$(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip'
               deployMethod: 'zipDeploy'
+          - script: |
```

Comment:
We need to check here if the project uses "goose" or not. Because we can't obligate everybody start using it.